lib/fft.d.ts

```diff
@@ -0,0 +1,26 @@
+export = FFT;
+declare function FFT(size: any): void;
+declare class FFT {
+    constructor(size: any);
+    size: number;
+    _csize: number;
+    table: any[];
+    _width: number;
+    _bitrev: any[];
+    _out: any;
+    _data: any;
+    _inv: number;
+    fromComplexArray(complex: any, storage: any): any;
+    createComplexArray(): any[];
+    toComplexArray(input: any, storage: any): any;
+    completeSpectrum(spectrum: any): void;
+    transform(out: any, data: any): void;
+    realTransform(out: any, data: any): void;
+    inverseTransform(out: any, data: any): void;
+    _transform4(): void;
```

Comment:
I don't think we need to have declarations for any internal methods or fields (anything that starts with `_`). Would you agree?

Reply:
Definitely- thanks!